cmd/admin-bucket-remote-add.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/minio/minio/pkg/auth"
 	"github.com/minio/minio/pkg/console"
 	"github.com/minio/minio/pkg/madmin"
+	"k8s.io/apimachinery/pkg/api/resource"
```

Comment:
I think we should use humanize package which we standardized for `mc` @kerneltime

Reply:
Yup, moving back to humanize after discussion, as long as humanize is a superset of what kubernetes resources use for documentation it should be fine.